requirements-dev.txt

```diff
@@ -1,2 +1,4 @@
 black==19.10b0
+blackadder
 eth-brownie>=1.11.10,<2.0.0
+pre-commit
```

Comment:
If we can find some way to avoid precommit here, that'd be awesome. We have another JS-based precommit script too (which I would prefer to get rid of TBH), not sure how well they'd play together

Reply:
We can get rid of pre-commit. I guess something like this https://medium.com/better-programming/automatically-execute-bash-commands-on-save-in-vs-code-7a3100449f63 could be used to automatically format Vyper files on save using VSCode

Reply:
These vs code`settings.json` work fine for me locally (i.e. they format vyper files automatically on save). If you want I can add this to the README as an alternative to using `pre-commit`
```
"emeraldwalk.runonsave": {
	"commands": [
		{
			"match": "\\.vy$",
			"cmd": "blackadder --fast '${file}'"
        }
	]
}
```

Reply:
Yes, and I use neovim, so I have another way to add this to my own editor as well.

I usually like things that affect people's dev environments to not be committed to the repository itself.